cspell.yml

```diff
@@ -187,4 +190,4 @@ words:
   - xtask
   - zbuild
   - zeroize
-  - zunstable
+  - zunstable
```

Comment:
@berlin-with0ut-return, a few files have the empty new line removed at the end of the file. Can you please ensure each file has one before completing?